lib/Extension/CodeTransformExtra/Rpc/ImportClassHandler.php

```diff
@@ -74,12 +74,13 @@ public function configure(Resolver $resolver)
     public function handle(array $arguments)
     {
         if (null === $arguments[self::PARAM_QUALIFIED_NAME]) {
-            $suggestions = $this->suggestions($arguments[self::PARAM_NAME]);
+            $name = (new WordAtOffset('\s|%|\(|\)|\[|\]|:|;|\r|\r\n|\n'))($arguments[self::PARAM_SOURCE], $arguments[self::PARAM_OFFSET]);
```

Comment:
This utility is from `phpactor/text-document`, this regex is the default split pattern without the `\` char. It would probably make sense to add this pattern as a constant in that utility.

Reply:
Could now use `WordAtOffset(WordAtOffset::SPLIT_QUALIFIED_PHP_NAME)`